src/kirby/components/modal/config/modal-config.ts

```diff
@@ -0,0 +1,7 @@
+export interface ModalConfig {
+  title: string;
+  titleHorizontalAlignment?: 'left' | 'center';
+  closeIconName?: 'close' | 'arrow';
```

Comment:
`kirby-icon` currently has no support for `arrow`, consider using `arrow-back` for now (or else it will render a coq wheel)
```suggestion
  closeIconName?: 'close' | 'arrow-back';
```